docs/mkdocs.yml

```diff
@@ -72,6 +72,7 @@ nav:
       - ClickHouse: https://clickhouse.com/docs/en/engines/table-engines/integrations/iceberg
       - Daft: daft.md
       - Databend: https://docs.databend.com/guides/access-data-lake/iceberg
+      - Debezium: https://memiiso.github.io/debezium-server-iceberg/
```

Comment:
what is `memiiso`? is this related to https://debezium.io/?

Reply:
This GitHub organization im publishing open-source projects under it. It is not official Debezium org

Reply:
@nastra do you think `Memiiso Debezium` or `Memiiso Debezium Project` be suitable? what do you think?

Reply:
renamed the link to `Memiiso Debezium`